Using-GTSAM-EXPORT.md

```diff
@@ -0,0 +1,41 @@
+# Using GTSAM_EXPORT
+
+To create a DLL in windows, the `GTSAM_EXPORT` keyword has been created and needs to be applied to different methods and classes in the code to expose this code outside of the DLL.  However, there are several intricacies that make this more difficult than it sounds.  In general, if you follow the following three rules, GTSAM_EXPORT should work properly.  The rest of the document also describes (1) the common error message encountered when you are not following these rules and (2) the reasoning behind these usage rules.
```

Comment:
> the `GTSAM_EXPORT` keyword has been created 

Should it be clearer mentioning that it is CMake the one who automatically defines this symbol (as far as I know) when building dynamic libs?